modules/nf-core/chromap/chromap/main.nf

```diff
@@ -9,8 +9,8 @@ process CHROMAP_CHROMAP {
 
     input:
     tuple val(meta), path(reads)
-    path fasta
-    path index
+    tuple val(meta2), path (fasta)
```

Comment:
nitpicking here, but shouldn't meta2 == meta3 here? I assume the aligner index is based on the the fasta file?

Reply:
yes, it's the same, what do you suggest that we get rid of the meta of the `fasta` file?

Reply:
I tend to use the same variable for both, since they should be the same. Not that it matters much in to module, but it does indicate we expect it to be the same.

Reply:
👉 [cfdc086](https://github.com/nf-core/modules/pull/2416/commits/cfdc086f72344be9051b09903af043efbdd41660)

Reply:
OK, I have now used the same variable 👉 [1fc5a26](https://github.com/nf-core/modules/pull/2416/commits/1fc5a26ad04a6e7c64ace56f3eb29003d88d77a0)